Automattic/wp-calypso
# Update Search upsell SVG: Magnify icon is reversed.

#### Proposed Changes

This PR updates the Jetpack Search upsell SVG image.  The magnify icon in the illustration should be facing forward to the right, same as it is in the Search **Icon**. Prior to this PR, the magnify icon was facing back to the left.

Related to: 1164141197617539-as-1202234182051627/f

#### Screenshots:

**Before**                                 |  **After**
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/11078128/172892955-a8f5afd6-7391-4b36-a228-3e933bf2059a.png)  |  ![](https://user-images.githubusercontent.com/11078128/172892976-e7c510b2-098b-4813-a500-59540b2af29b.png)

#### Testing Instructions

- Checkout this PR and `yarn start`. **(Or use the **Calypso.live** direct link below.)**
- Select a Jetpack site that does not have an Jetpack Search subscription (or does not have a Plan with Search included).
- Navigate to _Jetpack_ --> _Search_.    (`/jetpack-search/:site`).
- Verify the Search upsell illustration looks the same, except that the magnify glass icon in the illustration is now facing forward to the right. (like shown in the "After" screenshot above.)
